models/user/user_system.go

```diff
@@ -36,9 +36,9 @@ func (u *User) IsGhost() bool {
 }
 
 const (
-	ActionsUserID    = -2
-	ActionsUserName  = "gitea-actions"
-	ActionsUserEmail = "teabot@gitea.io"
+	ActionsUserID    int64 = -2
+	ActionsUserName        = "gitea-actions"
+	ActionsUserEmail       = "teabot@gitea.io"
```

Comment:
Should we update to something like "gitea-actions@gitea.com"?

Reply:
I do not think so, actually it should be a non-deliverable email address, and it shouldn't use any official domain.